packages/sdk/src/rtl/authSession.ts

```diff
@@ -163,7 +162,7 @@ export abstract class AuthSession implements IAuthSession {
       return result.value
     } else {
       if (result instanceof Buffer) {
-        throw sdkError({message: result.toString(utf8)})
+        throw sdkError({message: result.toString('utf8')})
```

Comment:
I see constants for 'utf8' elsewhere. Why the change?

Reply:
importing and cross-package dependency issues. Seemed heavy handed after monorepo work.

Reply:
Also, this is the only utf8 ref in that source file, so I didn't think a const added value. Happy to refactor this in the future as part of the evolving monorepo work if that's a code smell to you. I think 
```typescript
export const utf8encoding = {encoding: 'utf-8'}
```
might be the const to import in those scenarios in that case

Reply:
I'm okay with this. It was an observation more than anything else